pkg/dynamiclistener/server.go

```diff
@@ -596,3 +646,35 @@ func (ln tcpKeepAliveListener) Accept() (c net.Conn, err error) {
 	tc.SetKeepAlivePeriod(3 * time.Minute)
 	return tc, nil
 }
+
+func lookupTLSVersion(stringMinTLSVersion string) (uint16, error) {
+	tlsVersionKeys := getKeysFromMap(tlsVersions)
+	if val, ok := tlsVersions[stringMinTLSVersion]; ok {
+		return val, nil
+	}
+	return 0, fmt.Errorf("Invalid minimal TLS version [%s], must be one of: %s", stringMinTLSVersion, strings.Join(tlsVersionKeys, " "))
+}
+
+func lookupTLSCiphers(stringTLSCiphers string) ([]uint16, error) {
+	tlsCipherSuitesKeys := getKeysFromMap(tlsCipherSuites)
+	sliceTLSCiphers := strings.Split(stringTLSCiphers, ",")
```

Comment:
Should support whitespace (`cipher1, cipher2`)

Reply:
Each found cipher will now be stripped from leading and trailing spaces before being validated.